app/src/main/java/org/metabrainz/android/presentation/features/userprofile/ProfileActivity.kt

```diff
@@ -0,0 +1,104 @@
+package org.metabrainz.android.presentation.features.userprofile
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.google.android.material.tabs.TabLayoutMediator
+import org.metabrainz.android.R
+import org.metabrainz.android.databinding.ActivityProfileBinding
+import org.metabrainz.android.presentation.features.dashboard.DashboardActivity
+import org.metabrainz.android.presentation.features.login.LoginSharedPreferences
+import org.metabrainz.android.presentation.features.settings.SettingsActivity
+
+class ProfileActivity : AppCompatActivity() {
+    lateinit var binding: ActivityProfileBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding= ActivityProfileBinding.inflate(layoutInflater)
```

Comment:
Since this is a new activity altogether, we can look to omit this style of declaration and avoid using XML files.

Reply:
deleted playeractivity xml, and cleared related binding in ProfileActivity.kt